tests/Dart/Tests.Util.fs

```diff
@@ -0,0 +1,15 @@
+module Fable.Tests.Dart.Util
+
+open Fable.Core
+
+type Assertion<'T> =
+    interface end
+
+[<ImportAll("package:test/test.dart")>]
```

Comment:
That's pretty cool! I'm wondering if we can fork the [dartdoc](https://github.com/dart-lang/dartdoc/) package to get the public api and generate bindings for any other package or even read the `pubspec.yaml` and generate a file with bindings for all the current package dependencies.

Reply:
Thanks for checking @Nhowka! I think that'd be great idea. I'm still learning Dart so if someone with more experience could have a look at forking dartdoc that'd be awesome.

Note that bindings for Dart will likely be slightly different than in JS. For JS we usually define an interface and then assign it to a value. In Dart it's not possible to assign a module to a value, so I assume we'll write the bindings usually as a static class with dummy member bodies (`nativeOnly`) as in here. We've also discussed about doing the same in JS but there're already many bindings written with the "interface style".

Reply:
> we'll write the bindings usually as a static class with dummy member bodies (nativeOnly) as in here. We've also discussed about doing the same in JS but there're already many bindings written with the "interface style".

@alfonsogarciacaro
Can you give an example or point to the discussion, we're looking for the best representation of ad-hoc bindings for Rust too.

Reply:
As usual the discussion is scattered in different places, but I think there was a summary here: https://github.com/fable-compiler/fable-browser/issues/77

Reply:
I'll take a shot on dartdoc, maybe generating a JSON representation with all the surface might be reusable on that side and we can consume it to generate our bindings.

Would it be feasible to attribute an entire module with an `Import`? I'm wondering that because as Dart has top-level methods and fields besides the instance ones, so maybe we could leverage that to make a better 1:1 representation of the api. Also as Dart has no inner classes nor overloads, top level functions and values would be the static top-level methods and fields, and types and/or interfaces would represent the classes. For the generated file, the import could also have the same name as the module, like

```fs
[<Import("package:some_package/some_file.dart")>]
module SomeFile = ...
```
turning into
```dart
import "package:some_package/some_file.dart" as SomeFile;
```

Dart is kinda quirky as every class is also an interface that can be implemented, it can be somewhat useful but I'm not sure how we could or even should represent that... Named constructors and factories could probably be encoded as static methods returning the class without any issues.

Reply:
That's a good appreciation @Nhowka! Though I think it may be a bit difficult. Let's move the discussion to a separate issue so we can share the experience with other languages 👍 #2779